app/models/log.rb

```diff
@@ -1,3 +1,7 @@
 class Log < ApplicationRecord
   serialize :supporting_details
+  has_many :user_logs, dependent: :destroy
+  has_many :users, through: :user_logs
+
+  default_scope { order('created_at desc') }
```

Comment:
you can set the default order of the table here. (`order(created_at: :desc)`)
so it will use everywhere you call the logs record